### PR TITLE
Avoid reading active bindings when printing

### PR DESCRIPTION
**Pull request by oscardelama**
Including the fix and extending a test case

### CODE REVIEW

File: tests/testthat/test-portable.R [hunk 1]
```diff
@@ -112,6 +112,18 @@ test_that("Active bindings work", {
...
         else self$x <- value/2
+      },
+
+      sqrt_of_x = function(value) {
+        if (!missing(value))
+          # In "setter" role
+          stop("Sorry this is a read-only variable.")
+        else {
+          # In "getter" role
+          if (x < 0) stop("The requested value is not available.")
```
Comment (wch) [inline tests/testthat/test-portable.R]: Should these be `self$x`?

File: R/print.R [hunk 2]
```diff
@@ -70,12 +70,14 @@ object_summaries <- function(x) {
     obj_names <- ls(x, all.names = TRUE)
 
   values <- vapply(obj_names, function(name) {
-    obj <- x[[name]]
     if (is.environment(x) && bindingIsActive(name, x)) "active binding"
```
Comment (wch) [inline R/print.R]: I'd prefer that this has a style like:

```R
if (is.environment(x) && bindingIsActive(name, x)) {
  "active binding"
} else {
  ...
}
```

**Comment by wch** [comment]
Thanks! I have just a few minor comments.

**Comment by oscardelama** [comment]
yeah, *there's more than one way to skin a cat* (ugly proverb). Anyway, I am glad you like it.

**Comment by wch** [comment]
@oscardelama Once the comments are addressed (you can add another commit and push), I'll merge this PR.

**Comment by oscardelama** [comment]
Oops. Sorry if I look rather slow. I am a noob in this matters. This is my first pull request and I didn't figure out what was the next step. 

I understand I should align my fork with your comments and that's it, Am I right?

**Comment by gaborcsardi** [comment]
@oscardelama Update your code, commit and push. That's all you need to do.

**Comment by oscardelama** [comment]
OK guys. Its ready.

**Comment by wch** [comment]
Thanks!

**Comment by oscardelama** [comment]
Thanks to you!

**Pull request merged**